services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -304,16 +319,20 @@ def get_dynamic_sidecar_spec(
             dynamic_services_scheduler_settings.DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS,
             scheduler_data.callbacks_mapping,
         )
-        | StandardSimcoreDockerLabels(
-            user_id=scheduler_data.user_id,
-            project_id=scheduler_data.project_id,
-            node_id=scheduler_data.node_uuid,
-            product_name=scheduler_data.product_name,
-            simcore_user_agent=scheduler_data.request_simcore_user_agent,
-            swarm_stack_name=dynamic_services_scheduler_settings.SWARM_STACK_NAME,
-            memory_limit=ByteSize(0),  # this should get overwritten
-            cpu_limit=0,  # this should get overwritten
-        ).to_simcore_runtime_docker_labels(),
+        | standard_simcore_docker_labels
+    )
+
+    # add autoscaling constraints if pricing plan is required
+    if hardware_info and len(hardware_info.aws_ec2_instances) == 1:
```

Comment:
Q; wondering about that check of len == 1.
not sure it is good to have that here. not sure it is good to have no error coming in the logs in case it is bigger.

At some point we might want to have backup machines. so maybe I would suggest to at least write a warning that putting more than one is currently unsupported. otherwise we might wonder what is going on